### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by phase1geo**
Includes the following projects:

- Outliner
- TextShine
- Actioneer
- Annotator
- Journaler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,6 +248,7 @@ this list aims to be broader and include apps from various other ecosystems in v
 
 #### Drawing & Editing
 
+- [Annotator](https://github.com/phase1geo/annotator) - Communicate with images. Better. `#vala` `#granite`.
```
Comment (loicreynier) [inline README.md]:
```suggestion
- [Annotator](https://github.com/phase1geo/annotator) - Image annotation application designed for elementary OS `#vala` `#granite`.
```

File: README.md [hunk 2]
```diff
@@ -475,6 +478,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 
 ### Desktop Productivity
 
+- [Actioneer](https://github.com/phase1geo/actioneer) - Automate actions on file changes `#vala` `#granite`.
```
Comment (loicreynier) [inline README.md]:
```suggestion
- [Actioneer](https://github.com/phase1geo/actioneer) - Tool to automate actions on file changes `#vala` `#granite`.
```

**Review by loicreynier — CHANGES_REQUESTED** [review]
LGTM. I made some slight rephrasing to better align with the usual style employed here.

File: README.md (hunk 2 above)
Comment (phase1geo) [reply README.md]: I am good with these changes.

**Review by loicreynier — APPROVED** [review]

**Pull request merged**